configure.ac

```diff
@@ -293,6 +294,10 @@ AC_CHECK_LIB(json-c, json_object_get_uint64, [AC_DEFINE([JSONC_UINT64_SUPPORT],
     [then]
         AC_CHECK_HEADERS([xenstore.h xs.h])
         AC_DEFINE([HAVE_LIBXENSTORE], [1], [Define to 1 to enable Xenstore support.])
+        [if test "$have_zlib" = "yes"]
+        [then]
+            AC_DEFINE([HAVE_ZLIB], [1], [Define to 1 to enable QCOW2 disk parsing.])
```

Comment:
`PKG_CHECK_MODULES` should already define in config.h whether zlib is found, you don't need to do this

Reply:
I've tried to remove explicit defininition, but zlib didn't appear in config.h

Reply:
Ah yes sorry it sets the cflags and libs variables not config.h. Still, you could just define HAVE_ZLIB in the PKG_CONFIG_CHECH line:

```
PKG_CHECK_MODULES([ZLIB], [zlib],
   [AC_DEFINE([HAVE_ZLIB], [1], [Found ZLIB])],[])